### PR TITLE
Make SlackChannelId non optional

### PR DESCRIPTION
**Pull request by leogrim**
@ryanpbrewster

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/slack/models/SlackIncomingWebhookInfo.scala [hunk 1]
```diff
@@ -121,12 +117,10 @@ class SlackIncomingWebhookInfoRepoImpl @Inject() (
   private def workingRows = activeRows.filter(row => row.lastFailure.isEmpty)
   def table(tag: Tag) = new SlackIncomingWebhookInfoTable(tag)
   initTable()
-  override def deleteCache(info: SlackIncomingWebhookInfo)(implicit session: RSession): Unit = {
-    info.slackChannelId.foreach(channelId => integrationsCache.remove(SlackChannelIntegrationsKey(info.slackTeamId, channelId)))
-  }
-  override def invalidateCache(info: SlackIncomingWebhookInfo)(implicit session: RSession): Unit = deleteCache(info)
+  override def deleteCache(info: SlackIncomingWebhookInfo)(implicit session: RSession): Unit = {}
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/models/SlackIncomingWebhookInfo.scala:120]: Intentionally not invalidating cache?

**Comment by ryanpbrewster** [comment]
:triumph:

**Comment by ryanpbrewster** [comment]
You're my hero :swoon:

File: kifi-backend/modules/shoebox/app/com/keepit/slack/models/SlackIncomingWebhookInfo.scala (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/slack/models/SlackIncomingWebhookInfo.scala:120]: Yep actually that cache hasn't anything to do with that webhooks in the first place

**Pull request merged**